frappe/integrations/utils.py

```diff
@@ -10,15 +10,15 @@
 from frappe.utils import get_request_session
 
 
-def make_request(method, url, auth=None, headers=None, data=None, json=None):
+def make_request(method, url, auth=None, headers=None, data=None, params=None, json=None):
```

Comment:
```suggestion
def make_request(method, url, auth=None, headers=None, data=None, json=None, params=None):
```

New arguments should always be at end.

---

TBH this function is VERY redundant. An abstraction over already decent abstraction - the `requests` module.

Reply:
Maybe make_request should be replaced with vanilla request?

Reply:
Only difference is automatic raising of exception? That's better done by adding hooks to request session then wrapping entire code like this.

Reply:
Oh yes! So true, please get rid of this. It took me a +- 1/2 day to figure out that it does little more than nothing and, on top of all, gets into ones way.